src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbsContainer = document.getElementById('thumbs');
+
+// eslint-disable-next-line no-shadow
+thumbsContainer.addEventListener('click', (event) => {
+  const link = event.target.closest('a');
```

Comment:
The use of `event.target.closest('a')` is correct for finding the closest anchor element. However, ensure that the HTML structure supports this logic, meaning the thumbnails should be wrapped in anchor tags. If the structure changes, this logic might break.